File: SEMESTER 1/PYTHON PROGRAMMING LAB/EXPERIMENT 6/QUESTION 1.py
d=dict()


for i in range(3):
    num = int(input("Enter the number of values to be inserted: "))  
    for j in range(n):
        val = int(input("Enter a number: "))  
        if val in d.keys():
            d[val]+=1
        else:
            d[val]=1
            
            
print("the number of times each value has occurred: ")


for i in d.keys():
    print(i,": ",d[i])
    
    
    

File: SEMESTER 1/PYTHON PROGRAMMING LAB/EXPERIMENT 8/QUESTION 1.c
def max_min(data):
  l = data[0]
  s = data[0]
  for num in data:
    if num> l:
      l = num
    elif num< s:
        s = num
  print("largest:",l,"\n smallest:",s)


n=int(input("enter the number of input:"))
lst=[]

print("enter input:")

for i in range(n):
    lst.append(input("enter the input:"))
print(max_min(lst))
